plugins/CoreHome/Tracker/LogTable/Action.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
In theory this and `LinkVisitAction` should be defined in `Actions` plugin. `Conversion` and `ConversionItem` should be specified in `Goals` plugin. Problem is when disabling the actions plugin these LogTables would not be found anymore and segmenting by an action segment would break. This is because the plugins don't define proper dependencies to each other and not all segments are defined in the correct plugin. Therefore they are for now all in CoreHome plugin which cannot be disabled